### ISSUE TITLE
Radio buttons "role" incorrectly set at "group" - should be "radiogroup"

### ISSUE THREAD

**Issue by ouq77** [ng-bootstrap/ng-bootstrap#3084]
### Bug description:
Radio buttons have the role set incorrectly as "group" when it should be "radiogroup" according to [ARIA17](https://www.w3.org/TR/WCAG20-TECHS/ARIA17.htm).

![Screenshot from 2019-03-24 19-13-09](https://user-images.githubusercontent.com/1796191/54875771-d0048300-4e69-11e9-8630-bb694b5940a1.png)

### Versions of Angular, ng-bootstrap and Bootstrap:
Angular: 7.2.4
ng-bootstrap: 4.0.0
Bootstrap: 4.3.1
